x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/180_match_operator.yml

```diff
@@ -0,0 +1,192 @@
+---
+setup:
+  - requires:
+      cluster_features: [ "gte_v8.16.0" ]
+      reason: "Match operator added in 8.16.0"
+      test_runner_features: allowed_warnings_regex
+  - do:
+      indices.create:
+        index:  test
+        body:
+          mappings:
+            properties:
+              content:
+                type: text
+              id:
+                type: integer
+  - do:
+      bulk:
+        index: "test"
+        refresh: true
+        body:
+          - { "index": { } }
+          - { "content": "This is a brown fox", "id": 1 }
+          - { "index": { } }
+          - { "content": "This is a brown dog", "id": 2 }
+          - { "index": { } }
+          - { "content": "This dog is really brown", "id": 3 }
+          - { "index": { } }
+          - { "content": "The dog is brown but this document is very very long", "id": 4 }
+          - { "index": { } }
+          - { "content": "There is also a white cat", "id": 5 }
+          - { "index": { } }
+          - { "content": "The quick brown fox jumps over the lazy dog", "id": 6 }
+
+---
+"simple where match":
+  - do:
+      allowed_warnings_regex:
+        - "No limit defined, adding default limit of \\[.*\\]"
+      esql.query:
+        body:
+          query: 'FROM test | WHERE content MATCH "fox" | KEEP id | SORT id'
```

Comment:
This is going to fail without a capability, I think.

Reply:
Wait. You check the cluster feature already. I think we're better off checking capabilities. We're just adding too many things to do it all with features.

Reply:
This has been updated to use a capability. 👍